docs/content/features/exploiters/wmi.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "WMI Exploiter"
+draft: false
+description: "Exploits WMI"
+tags: ["exploiter", "wmi", "brute force"]
+pre: "<i class='fa fa-toolbox'></i> "
+---
+
+## WMI
+
+[WMI (Windows Management Instrumentation)](
+https://learn.microsoft.com/en-us/windows/win32/wmisdk/wmi-start-page) is a set
+of tools and extensions in Windows for managing and querying system information,
+configurations, and operations. Administrators use WMI to manage local and
+remote environments by monitoring system health, automating administrative
+functions, managing network resources, etc.
```

Comment:
Same as SMB. some background about how/why that allows you to execute commands would be good.